src/Command/InitializeCommitteeMembershipEventsCommand.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\CommitteeMembership;
+use AppBundle\Entity\Reporting\CommitteeMembershipAction;
+use AppBundle\Entity\Reporting\CommitteeMembershipEvent;
+use AppBundle\Referent\ManagedAreaUtils;
+use AppBundle\Repository\CommitteeRepository;
+use AppBundle\Repository\ReferentTagRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class InitializeCommitteeMembershipEventsCommand extends Command
```

Comment:
it's a command one shot,
I think we will be able to do the same stuff in one query sql after the merge of:
- my PR with Silence Repub which adds Committee relation with Membership
- @Remg PR which adds Tags for Committee

Reply:
Ok I put this PR aside for now. And we will see how to manage that after yours.